internal/services/compute/disk_encryption_set_resource.go

```diff
@@ -144,6 +144,7 @@ func resourceDiskEncryptionSetCreate(d *pluginsdk.ResourceData, meta interface{}
 					ID: utils.String(keyVaultDetails.keyVaultId),
 				},
 			},
+			RotationToLatestKeyVersionEnabled: &rotationToLatestKeyVersionEnabled,
```

Comment:
Why'd you add this?

Reply:
This is to set the value during the creation